src/components/FormGroup.tsx

```diff
@@ -0,0 +1,47 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+import "lbh-frontend/lbh/objects/_form-group.scss/";
+
+/**
+ * The proptypes for {@link FormGroup}.
+ */
+export interface FormGroupProps {
+  id?: string;
+  className?: string;
+  /**
+   * Whether there is an error in the form group or not.
+   */
+  error?: boolean;
```

Comment:
I suspect we might want to change this to be `string[]` or something when we hook up the error handling properly, but it's fine for now.